Realm/RLMSyncSubscription_Private.hpp

```diff
@@ -18,23 +18,51 @@
 
 #import "RLMSyncSubscription_Private.h"
 
+#import <string>
+
 namespace realm::sync {
 class Subscription;
 class SubscriptionSet;
 }
+namespace realm {
+class Query;
+}
 
 RLM_HEADER_AUDIT_BEGIN(nullability, sendability)
 
 @interface RLMSyncSubscription ()
 
 - (instancetype)initWithSubscription:(realm::sync::Subscription)subscription subscriptionSet:(RLMSyncSubscriptionSet *)subscriptionSet;
 
+@property (nonatomic, readonly) std::string_view stdString;
```

Comment:
This property name is just the type of the property and doesn't give any indication as to what it is.